test/forces.jl

```diff
@@ -159,3 +159,48 @@ end
 
     @test abs(diff_findiff - diff_forces) < 5e-4
 end
+
+@testitem "Forces match partial derivative of each term" setup=[TestCases] begin
+    using LinearAlgebra
+
+    function get_term_forces(lattice, atoms, positions; kgrid=[1,1,1], Ecut=4, symmetries=false, ε=1e-8)
+        model = model_DFT(lattice, atoms, positions; functionals=LDA(), symmetries, temperature=1e-3)
+        basis = PlaneWaveBasis(model; kgrid, Ecut)
+
+        scfres = self_consistent_field(basis; tol=1e-7)
+
+        map(1:length(basis.terms)) do iterm
+            test_atom = rand(1:length(model.atoms))
+            test_dir = randn(3)
```

Comment:
rand is better
Also for MPI tests to succeed you need to distribute randomness across processors (see other tests)

Reply:
Indeed, thanks!